src/components/AssetHeader/AssetActions.tsx

```diff
@@ -37,14 +40,26 @@ export const AssetActions = ({ isLoaded, assetId, accountId }: AssetActionProps)
       width={{ base: 'full', lg: 'auto' }}
     >
       <Skeleton isLoaded={isLoaded} width={{ base: 'full', lg: 'auto' }}>
-        <Tooltip label={translate('common.send')} fontSize='md' px={4} hasArrow>
-          <IconButton
-            onClick={handleSendClick}
-            isRound
-            width='full'
-            icon={<ArrowUpIcon />}
-            aria-label={translate('common.send')}
-          />
+        <Tooltip
+          label={
+            notEnoughCryptoToSell()
+              ? translate('common.insufficientFunds')
+              : translate('common.send')
+          }
+          fontSize='md'
+          px={4}
+          hasArrow
+        >
+          <div>
```

Comment:
The div needs to be here because when the direct child of the tooltip is disabled, then the tooltip is also disabled